RoslynPluginGenerator/SonarQube.Plugins.Roslyn.PluginGenerator.csproj

```diff
@@ -178,6 +180,20 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\RoslynProperties.java" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\sonar-roslyn-sdk-template-plugin-1.0-empty.jar" />
+  </ItemGroup>
+  <ItemGroup>
+    <COMReference Include="Shell32">
+      <Guid>{50A7E9B0-70EF-11D1-B75A-00A0C90564FE}</Guid>
+      <VersionMajor>1</VersionMajor>
+      <VersionMinor>0</VersionMinor>
+      <Lcid>0</Lcid>
+      <WrapperTool>tlbimp</WrapperTool>
+      <Isolated>False</Isolated>
+      <EmbedInteropTypes>True</EmbedInteropTypes>
+    </COMReference>
```

Comment:
The Shell.Folder class is a COM object. This ref is to the type library that defines it. *EmbedInteropTypes* is true, which means the required COM interface definitions will be embedded into our assembly which it is built.